src/devices/src/virtio/queue.rs

```diff
@@ -789,15 +821,29 @@ pub(crate) mod tests {
         let mut q = vq.create_queue();
         assert_eq!(vq.used.idx.get(), 0);
 
-        //index too large
-        q.add_used(m, 16, 0x1000);
-        assert_eq!(vq.used.idx.get(), 0);
+        //Valid queue addresses configuration
+        {
+            //index too large
+            q.add_used(m, 16, 0x1000).unwrap();
+            assert_eq!(vq.used.idx.get(), 0);
 
-        //should be ok
-        q.add_used(m, 1, 0x1000);
-        assert_eq!(vq.used.idx.get(), 1);
-        let x = vq.used.ring[0].get();
-        assert_eq!(x.id, 1);
-        assert_eq!(x.len, 0x1000);
+            //should be ok
+            q.add_used(m, 1, 0x1000).unwrap();
+            assert_eq!(vq.used.idx.get(), 1);
+            let x = vq.used.ring[0].get();
+            assert_eq!(x.id, 1);
+            assert_eq!(x.len, 0x1000);
+        }
+
+        //Invalid used ring address addresses
```

Comment:
Nit: Maybe `Invalid queue addresses configuration` would be better.

Reply:
Right, I will correct this.